File: EjerciciosEPI/1_sumatorio/sumatorio.c
#include <stdio.h>

int main () {
	int n;
	int numero_actual;
	int suma;

	suma = 0;
	printf("Introduzca el numero de numeros a introducir: ");
	scanf("%d", &n);

	for (int i = 0 ; i < n ; i++)
	{
		printf("Introduzca numero: ");
		scanf("%d", &numero_actual);
		suma+=numero_actual;
	}

	printf("El sumatorio es: %d", suma);
	
	return 0;
}

File: EjerciciosEPI/1_sumatorio/sumatorio.py
"""
Programa que muestre el sumatorio de n numeros. Todos ellos introducidos por el teclado.
n es un numero introducido por teclado tambien.
Ejemplo:
    Introduzca el numero de numeros a introducir: 5
    Introduzca numero: 1
    Introduzca numero: 7
    Introduzca numero: 0
    Introduzca numero: -1
    Introduzca numero: -2
    El sumatorio es: 5
"""

n = 0
numero_actual = 0
suma = 0

n = int(input("Introduzca el numero de numeros a introducir: "))

for i in range (0, n, 1):
    numero_actual = int(input("Introduzca numero: "))
    suma = suma + numero_actual

print("El sumatorio es:", suma)
